pkg/config/types.go

```diff
@@ -116,7 +116,7 @@ type Values struct {
 	BlockDeviceMappingSwap       string                            `mapstructure:"block-device-mapping-swap"`
 	ElasticInferenceAccelerator  types.ElasticInferenceAccelerator `mapstructure:"elastic-inference-accelerator"`
 	ElasticInferenceAssociations string                            `mapstructure:"elastic-inference-associations"`
-	EventID                      string                            `mapstructure:"event-id"`
+	EventId                      string                            `mapstructure:"event-id" json:"event-id"`
```

Comment:
the tag should be `json:"eventId"` and the property should be `EventID` unless I'm missing how this is processed.

Reply:
wow thanks for catching this! We don't have a test to validate, but maybe we should...

I tried your suggestion, but right now the output isn't capitalizing the 'e' in 'eventId' -- will take a look later this afternoon. Same for below comment

Reply:
oops should be `json:"EventId"`

Reply:
Addressed in latest commit